crates/matrix-sdk-base/src/store/ambiguity_map.rs

```diff
@@ -134,17 +136,17 @@ impl AmbiguityCache {
     fn update(
         &mut self,
         room_id: &RoomId,
-        old_map: Option<AmbiguityMap>,
-        new_map: Option<AmbiguityMap>,
+        old_map: Option<DisplayNameUsers>,
+        new_map: Option<DisplayNameUsers>,
     ) {
         let entry = self.cache.entry(room_id.to_owned()).or_default();
 
         if let Some(old) = old_map {
-            entry.insert(old.display_name, old.users);
+            entry.insert(old.display_name.clone(), old.users);
         }
 
         if let Some(new) = new_map {
-            entry.insert(new.display_name, new.users);
+            entry.insert(new.display_name.clone(), new.users);
```

Comment:
Are these two clones needed?

Reply:
Oh, indeed not, good spot: b5120457ec29364fed8d937098c8173bca1343e8.